Dart/src/com/jetbrains/lang/dart/analyzer/DartAnalysisServerService.java

```diff
@@ -169,6 +170,14 @@ public void computedNavigation(String file, List<NavigationRegion> regions) {
     }
 
     @Override
+    public void computedOverrides(String file, List<OverrideMember> overrides) {
+      synchronized (myOverrideData) {
+        myOverrideData.put(file, overrides);
+      }
+      forceFileAnnotation(file);
```

Comment:
When we switch to DAS driven features we'll get daemon restarted and caches dropped 4 times on each typing. I'm not sure it's good. Need to think about it.